Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionURLSchemeHandlerCocoa.mm

```diff
@@ -107,13 +109,15 @@
         }
 
         RefPtr<API::Error> error;
-        auto *fileData = extensionContext->extension().resourceDataForPath(requestURL.path().toString(), error);
-        if (!fileData) {
-            extensionContext->recordError(wrapper(error));
+        auto resourceData = extensionContext->extension().resourceDataForPath(requestURL.path().toString(), error);
+        if (resourceData.empty()) {
+            extensionContext->recordErrorIfNeeded(wrapper(error));
```

Comment:
On my local machine, this failed as recordError because no error was being passed. Using recordErrorAsNeeded lets it compile, but doesn't pass tests (they all timeout at 33 seconds locally).

Reply:
I'm not sure. I can try to build your branch locally sometime tomorrow and look into it.